python/doc/examples/probabilistic_modeling/stochastic_processes/plot_kronecker_covmodel.py

```diff
@@ -0,0 +1,71 @@
+"""
+Create a kronecker covariance model
+===================================
+"""
+# %%
+# This use case illustrates how the User can create a Kronocker covariance function, which allows defining a multivariate covariance function with a specific covariance structure.
+#  
+# For that purpose, a spatial covariance matrix :math:`\underline{\underline{C}}^{spat}` is needed in addition to a scalar correlation function :math:`\rho`.
+#
+#  We recall that model is defined by
+# .. math: :
+#
+#    C(\vect{s}, \vect{t}) = \rho\left(\dfrac{\vect{s}}{\theta}, \dfrac{\vect{t}}{\theta}\right) C ^ {stat}(\vect{s}, \vect{t}),
```

Comment:
I have the same remark as in the API doc file: does C^stat depend on s and t?

Reply:
No C^{stat} here does not depend on s,t